tezos/messages/src/p2p/encoding/deactivate.rs

```diff
@@ -5,10 +5,11 @@ use getset::Getters;
 use serde::{Deserialize, Serialize};
 
 use crypto::hash::ChainId;
+use tezos_encoding::enc::BinWriter;
 use tezos_encoding::encoding::HasEncoding;
 use tezos_encoding::nom::NomReader;
 
-#[derive(Serialize, Deserialize, Debug, Getters, Clone, HasEncoding, NomReader)]
+#[derive(Serialize, Deserialize, Debug, Getters, Clone, HasEncoding, NomReader, BinWriter)]
```

Comment:
@akoptelov do we still need Serialize/Deserialize for all these structs?

Reply:
I think we still use it for some messages to get JSON. But let me check that.

Reply:
Also it is used in ffi for some of the messages. I've removed `Deserialize` in some cases. Later we'll remove the rest. https://viablesystems.atlassian.net/browse/TE-693.